### DISCUSSION TITLE
dev branch Linux testing

### DISCUSSION THREAD

**Discussion by buzzmandt** [inkstitch/inkstitch#2235] [General]
I use inkstitch regularly (as well as YouTube tutorials) and I use Linux. I am offering to do testing of dev branch on Linux if it helps. I just need to know how and/or what to test. 

Currently running opensuse Tumbleweed.

**Comment by kaalleen** [comment]
Thanks for your offer to test recent Ink/Stitch builds. I think the best one to test the upcoming version with at the moment is this one here: https://github.com/inkstitch/inkstitch/releases/tag/dev-build-kaalleen-daniels-build

Other recent builds may have an issue with the simulator (black square).

Please note, that files created with this version are not backwards compatible. They cannot be used with older Ink/Stitch versions. Old files will update automatically when used with this version.

A more or less incomplete list of the changes can be found here: https://inkstitch.org/upcoming/

Any bug report or observation of unwanted behavior can be very helpful to make the next release better.

**Reply by buzzmandt** [reply]
Ok, just installed it. Do I report issues here or in the Issues section and make sure I mark it as daniels dev?

did the rpm and suse is complaining about it not being signed, simple to just ignore but might scare some people off.

also, I LOOOOVE some of what I'm seeing.

STOKED!!

screenshot suse complaint
![Screenshot_20230425_195944](https://user-images.githubusercontent.com/76489837/234433331-dd92b034-71cf-4b23-a5b6-7df280324703.png)

**Reply by kaalleen** [reply]
Yes, sure you can make videos :)

Preferably report issues in the issue section. A mention which dev release you are using would be fine. Usually this is already written into the error message you'l receive in Ink/Stitch.

Nice to hear that you enjoy the new features. We are so excited about the next release ourselves :)

**Reply by buzzmandt** [reply]
Excellent and outstanding 😁

I wish I could help more. Thank you much

**Reply by kaalleen** [reply]
You already do help a lot by helping people to understand Ink/Stitch better!
